extension/background.js

```diff
@@ -160,8 +160,8 @@ function sendData(name, action) {
 		'url' : data.url,
 		'os' : data.os,
 		'browser' : data.browser,
```

Comment:
What about these? Compared to:

https://github.com/retest/recheck-web/blob/6b9617670bc135a3e08cea8c0e9dc7aa41d90204/src/main/java/de/retest/web/meta/SeleniumMetadata.java

Not relevant here, but maybe we should also change the class name to `WebMetadata` as not everything is Selenium-specific.

Reply:
A bit more effort as browser currently is a summary of browser name and version. Same for os.

Reply:
Ah, OK. Thx for explaining.

Reply:
Changing the class name is a good idea. But not relevant before the release, right? It's internal anyways...

Reply:
Yes, was just a note. Nothing important.